conf/brs-default.properties

```diff
@@ -293,6 +293,15 @@ DEV.dumpPeersVersion =
 # Force re-build of derived objects tables at start.
 DEV.forceScan = off
 
+# Current public Testnet settings
+DEV.digitalGoodsStore.startBlock = 0
```

Comment:
Why add this in the default settings? It clutters it up, if we want to change this permanently, this should be removed from default properties and just changed in the Flux Values

Reply:
Good point, the idea was to make the settings on the *official* testnet the default. So there is no need to check on other places (other than turning the testnet flag on). If it is better to have it hard-coded in the Flux values, also fine by me. Just let me know and I update accordingly.